tracing/src/main/java/com/palantir/remoting3/tracing/Tracers.java

```diff
@@ -22,8 +22,6 @@
 
 /** Utility methods for making {@link ExecutorService} and {@link Runnable} instances tracing-aware. */
 public final class Tracers {
-    /** The key under which trace ids are inserted into SLF4J {@link org.slf4j.MDC MDCs}. */
-    public static final String TRACE_ID_KEY = "traceId";
```

Comment:
I don't think we should delete public static final fields like this, because if some consumer is depending on `com.palantir.remoting3.tracingTracers.TRACE_ID_KEY` then they'd suddenly break?

Reply:
it doesn't seem like anyone is using it, https://github.com/search?p=2&q=Tracers.TRACE_ID_KEY&type=Code
i also did a quick search on the key internally and  no internal consumer is using it.

Reply:
I still don't think we should remove things like this when there isn't a clear motivation - what if there was a user that didn't appear in your search?  Did you check OSS consumers too (e.g. atlasdb?)